pkg/logs/pipeline/provider.go

```diff
@@ -106,6 +110,53 @@ func (p *provider) Stop() {
 	p.outputChan = nil
 }
 
+func (p *provider) reconfigureSDS(config []byte, agentConfig bool) error {
+	var responses []chan error
+
+	typ := sds.StandardRules
+	if agentConfig {
+		typ = sds.AgentConfig
+	}
```

Comment:
Would it make sense to accept `typ` directly? This function is called from the same file, callers have access to the same constants.

Reply:
You're right 🤦 thanks.